report/main.qmd

```diff
@@ -1 +1,166 @@
+```
+
+```{r}
+library(tidyverse)
+#devtools::install_github("stat545ubc-2023/rmiss")
+library(rmiss)
+library(knitr)
+library(here)
+```
+
+## Introduction
+Sugary beverage consumption poses health concerns because of its association with several health challenges like obesity and diabetes. Different intervention methods are proposed to steer consumers toward zero-calorie beverages that are thought healthier. This study aims to investigate the efficacy of five different intervention methods, three of which focus on visual calorie messaging via posters, and the other two focus on price discount. To assess and compare different intervention methods, sales of zero-calorie beverages and sugary beverages under different interventions were monitored in three different sites. 
+
+The statistical questions of interest are: 
+
+1. Do the interventions significantly influence the sales of zero-calorie and sugary beverages?
+2. Which interventions are more effective? Specifically, does the combination of interventions lead to larger effects? Does presenting calorie-equivalent messaging have larger impacts than presenting the simple calorie admonishment?
+3. Does the effectiveness of different interventions vary between sites?
+
+The structure of the remainder of this report is as follows. Following a brief description about the dataset in Section 2, suggestions on exploratory data analysis (EDA) are presented in Section 3. Section 4 proposes formal analysis methods, and Section 5 summarizes our recommendations. 
+
+## Data Summary
+This is a quasi-experiment study for investigating the effects of five interventions  on consumer inclination towards zero-calorie beverages. The dataset contains the daily sales of different types of beverages (including zero-calorie, sugared and others) under various interventions across three sites, from October 27 to May 23. In this study, the daily sales of each type of beverage are used for measuring the consumers' preference on this type of beverage. The crucial variables are the daily sales of zero-calorie beverages, sugared beverages and total of all the beverages, which are measured as continuous data. On the other hand, based on the objectives of the study, the site and intervention should be treated as categorical variables for further analysis, and the days of week are also recorded as a categorical variable to determine potential impacts of time on the outcomes. Additionally, the dataset includes a time-counting variable that captures the timeline of the entire experiment, indicating that each site has around 220 time points. For the entire dataset, the records of these key variables are relatively complete, with sugared and zero-calorie beverages having nine observations missing. However, records for other types of beverages show a certain proportion of missing data, which could be because they are not the items of interest and excluded in recording.
```

Comment:
"quasi-experimental" instead of "quasi-experiment"
"preference for this type of beverage" instead of "preference on this type of beverage"
Give the total number of observations
Respecify how many levels of site and intervention there are
When mentioning other beverages, it may be informative to detail some of them